.github/workflows/update-nimbus-experiments.yml

```diff
@@ -37,3 +41,10 @@ jobs:
           title: "Update initial experiments JSON for Nimbus"
           body: "This (automated) PR updates the initial_experiments.json on the `main` branch"
           delete-branch: true
+      - name: Enable Pull Request Automerge
+        if: steps.create-pull-request.outputs.pull-request-operation == 'created'
+        uses: peter-evans/enable-pull-request-automerge@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          pull-request-number: ${{ steps.create-pull-request.outputs.pull-request-number }}
+          merge-method: squash
```

Comment:
I believe this is made unnecessary by #26902.  That also means all merge rules are kept in one place.

Reply:
Fair enough! I'll remove it from the github action then